core/src/main/java/bio/terra/pearl/core/service/notification/substitutors/EnrolleeEmailSubstitutor.java

```diff
@@ -45,6 +45,12 @@ protected EnrolleeEmailSubstitutor(EnrolleeContext ruleData, NotificationContext
         valueMap.put("siteMediaBaseUrl", getImageBaseUrl(contextInfo.portalEnv(), contextInfo.portalEnvConfig(), contextInfo.portal().getShortcode()));
         valueMap.put("siteImageBaseUrl", getImageBaseUrl(contextInfo.portalEnv(), contextInfo.portalEnvConfig(), contextInfo.portal().getShortcode()));
         valueMap.put("profile", enrolleeContext.getProfile());
+        if (enrolleeContext.getParticipantUser().getUsername().contains("-prox-")) {
```

Comment:
I couldn't find another way to glean this information from the enrollee context. I think this is probably OK but obviously not foolproof, so let me know if there's a better way to glean this information.